src/wallet3/rpc/omq_server.cpp

```diff
@@ -34,9 +34,9 @@ OmqServer::set_omq(std::shared_ptr<oxenmq::OxenMQ> omq_in, wallet::rpc::Config c
 {
   omq = omq_in;
 
-  //TODO: parametrize listening address(es) and auth
-  omq->listen_plain(std::string("ipc://./") + config.sockname);
+  omq->listen_plain(std::string("ipc://") + config.sockname);
```

Comment:
Prefer string literals:
```suggestion
  omq->listen_plain("ipc://"s + config.sockname);
```